### ISSUE TITLE
[Bug] Returning data in different types

### ISSUE THREAD

**Issue by czarekk215** [eclipse-zenoh/zenoh-backend-influxdb#267]
### Describe the bug

I switched from zenoh 0.11.0 to 1.0.0 version. I use influxdb v1 to store data from ros2 node. Ask Rest api to  get data from db returns valued in different formats like:
```
{"key":"sensor_publisher_temp","value":"aW52YWxpZCB1dGYtOCBzZXF1ZW5jZSBvZiAxIGJ5dGVzIGZyb20gaW5kZXggNzA=","encoding":"zenoh/bytes","timestamp":"7429687337599560704/ab85ed18b7c453d6849c2e1b759f2f57"},{"key":"sensor_publisher_temp","value":"aW52YWxpZCB1dGYtOCBzZXF1ZW5jZSBvZiAxIGJ5dGVzIGZyb20gaW5kZXggNzA=","encoding":"zenoh/bytes","timestamp":"7429687337599714432/ab85ed18b7c453d6849c2e1b759f2f57"},{"key":"sensor_publisher_temp","value":"aW52YWxpZCB1dGYtOCBzZXF1ZW5jZSBvZiAxIGJ5dGVzIGZyb20gaW5kZXggNzA=","encoding":"zenoh/bytes","timestamp":"7429687337599736448/ab85ed18b7c453d6849c2e1b759f2f57"},{"key":"sensor_publisher_temp","value":"AAEAAAUAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAABAAAAAQAAAAEAAAAAABRCAAAAAAAAAAABAAAAAAAAAAEAAAAAAAAAAQAAAAAAAAAAAAAAAAAAAAEAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAABAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA=","encoding":"zenoh/bytes","timestamp":"7429687346189489808/ab85ed18b7c453d6849c2e1b759f2f57"},{"key":"sensor_publisher_temp","value":"AAEAAAUAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAABAAAAAQAAAAEAAAAAABRCAAAAAAAAAAABAAAAAAAAAAEAAAAAAAAAAQAAAAAAAAAAAAAAAAAAAAEAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAABAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA=","encoding":"zenoh/bytes","timestamp":"7429687346189910368/ab85ed18b7c453d6849c2e1b759f2f57"},
```
On version 0.11.0 every row was the same format, because now values has different format it's hard to deserialize the in the sam way.

### To reproduce

1. Start influxdb v1
2. Start Zenoh server with zenoh-backend-ifnluxdb
3. Start zenoh-ros2-bridge
4. Start ros2 publisher 
5. Rest api ask to get data

### System info

- Platform Ubuntu 22.04
- Zenoh version 1.0.0

**Comment by mangodan2003** [comment]
It looks like this is basically the same as what I was about to report, though the description is unclear. Basically it seems that if you try to store binary data it gets incorrectly interpreted as utf-8 which fails, and the result is that the db holds a record that is a base64 encoded string saying that it failed to interpret the data as utf-8:

aW52YWxpZCB1dGYtOCBzZXF1ZW5jZSBvZiAxIGJ5dGVzIGZyb20gaW5kZXggNzA => "invalid utf-8 sequence of 1 bytes from index 70"

**Comment by mangodan2003** [comment]
Here's the culprit

https://github.com/eclipse-zenoh/zenoh-backend-influxdb/blob/main/v1/src/lib.rs#L446

Changing to this makes it work
            Err(err) => (true, b64_std_engine.encode(payload.to_bytes()).into()),

**Comment by mangodan2003** [comment]
This fix has just been merged into main : https://github.com/eclipse-zenoh/zenoh-backend-influxdb/commit/1095b3cfceecacd5d8ed2f8dd86ef19ac2de13cc

**Comment by JEnoch** [comment]
Fixed by #284. Thanks @mangodan2003 !
